packages/scss/src/components/card/component.scss

```diff
@@ -3,9 +3,11 @@
 	border-radius: var(--commons-borderRadius-L);
 	border-color: var(--commons-divider-color);
 	border-width: 1px;
```

Comment:
Why not use a box-shadow rather than a border?

Reply:
On this project, a large amount of components needed to be updated, so I edited only the essential to limit the risks. As border property is working fine and might be overrided on some cases, it seems safer to keep it as it is for now.